src/main/java/cook/Cook.java

```diff
@@ -0,0 +1,154 @@
+package cook;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+import java.util.HashMap;
+
+import exceptions.InvalidCommandException;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.ToDo;
+
+/**
+ * Cook class to store main logic and program.
+ */
+public class Cook {
+    private Storage storage;
+    private TaskList tasks;
```

Comment:
I like how you used plural form for your TaskList object since it's a collection of Tasks, good job!